packages/client/src/sigstore.ts

```diff
@@ -81,6 +81,31 @@ export async function verify(
   return verifier.verify(deserializedBundle, opts, payload);
 }
 
+export interface BundleVerifier {
+  verify(bundle: sigstore.SerializedBundle): void;
+}
+
+export async function createVerifier(
```

Comment:
I like this interface! A async factory function seems like a nice and simple way to handle the initialisation.